packages/web-components/fast-foundation/src/combobox/combobox.ts

```diff
@@ -497,30 +502,6 @@ export class Combobox extends FormAssociatedCombobox {
         }
     }
 
-    /**
-     * Handle keyup actions for value input and text field manipulations.
-     *
-     * @param e - the keyboard event
-     * @internal
-     */
-    public keyupHandler(e: KeyboardEvent): boolean | void {
```

Comment:
@radium-v, I removed this because I don't think it really does anything. If anything, it seems to be unnecessary overhead, as we will filter twice for certain keystrokes like `Delete` and `Backspace`. I'm not sure why we would want to set `selectedIndex` to -1 for any of these key strokes. To me, the `inputHandler` is sufficient for doing the filtering and setting `selectedIndex` to the appropriate value.

Reply:
This is needed for autocomplete `inline` and `both` functionality to work properly. When navigating through a filtered list after typing text, the auto-filled content is automatically highlighted. Pressing any of these keys will cause the input selection to change, which changes the autocomplete comparison value.

Reply:
Could we at least remove the Backspace and Delete cases (as these should be handled by the `inputHandler`)?